iphone/Classes/TiUITabGroup.m

```diff
@@ -49,6 +49,14 @@ - (UITabBarController *)tabController
       appearance.backgroundColor = UIColor.clearColor;
       controller.tabBar.scrollEdgeAppearance = appearance;
     }
+
+    // opt-out new iPadOS 18+ floating tab bar
+    if ([TiUtils isIOSVersionOrGreater:@"18.0"] && [TiUtils isIPad]) {
+      BOOL optOutTopTabBar = [TiUtils boolValue:[self.proxy valueForUndefinedKey:@"optOutTopTabBar"] def:NO];
```

Comment:
rename the property to `forceBottomPosition`, as tab bars are named tab groups in Titanium and opting out may mean something else in future API contexts.

Reply:
Aside that, you should really update your app code, as this will not become better in the future.